components/project-card/project-card.jsx

```diff
@@ -294,7 +385,8 @@ ProjectCard.propTypes = {
 };
 
 ProjectCard.defaultProps = {
-  onDetailView: false
+  onDetailView: false,
+  moderationState: `` // id of the moderation
```

Comment:
if this is an id, having it be a string by default is kind of weird. Why not `false` or `undefined`?